code/controllers/subsystem/speech_controller.dm

```diff
@@ -0,0 +1,43 @@
+SUBSYSTEM_DEF(speech_controller)
+	name = "Speech Controller"
+	wait = 1
+	flags = SS_TICKER
+	priority = FIRE_PRIORITY_SPEECH_CONTROLLER//has to be high priority, second in priority ONLY to SSinput
+	init_order = INIT_ORDER_SPEECH_CONTROLLER
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+
+	///list of the form: list(client mob, message that mob is queued to say, other say arguments (if any)).
```

Comment:
You shouldn't need to support other say args I don't think. Whisper() is a generalized proc, we only care about whispers that happen as the result of a verb

Reply:
now they simply call the relevant verb that the message was queued for (options are say(), whisper(), and emote())